trade-finance-manager-api/src/constants/payloads.js

```diff
@@ -1,49 +0,0 @@
-const BANK = {
-  id: String,
-  name: String,
-  mga: Object,
-  emails: Object,
-  companiesHouseNo: String,
-  partyUrn: String,
-  hasGefAccessOnly: Boolean,
-  paymentOfficerTeam: Object,
-};
```

Comment:
This BANK object didn't have
```
  utilisationReportPeriodSchedule: 'Object',
  isVisibleInTfmUtilisationReports: 'Boolean',
```
Somewhat surprising that old or new implementation didnt throw any errors